vivisect/qt/funcgraph.py

```diff
@@ -342,6 +342,7 @@ def refresh(self):
         after every change.  
         '''
         self._last_viewpt = self.mem_canvas.page().mainFrame().scrollPosition()
+        # FIXME: history should track this as well and return to the same place
```

Comment:
I like this idea. Can you file an issue expanding on this a little more so we don't forget?

Reply:
i'll do ya one better.  

https://github.com/vivisect/vivisect/pull/264

it's a WIP, but starts to tear at the flesh of what i'm getting at.  
basic functionality seems to work ok.  i'm wondering whether it wants to track history like browsers, where you can go backwards then forwards again.

Reply:
:)